antenna/configlib.py

```diff
@@ -91,6 +101,22 @@ def parse_bool(val):
     raise ValueError('%s is not a valid bool value' % val)
 
 
+def parse_class(val):
+    """Parses a string, imports the module and returns the class
+
+    >>> parse_class('hashlib.md5')
+
+    """
+    module, class_name = val.rsplit('.', 1)
+    module = importlib.import_module(module)
+    try:
+        return getattr(module, class_name)
+    except AttributeError:
+        raise ValueError('%s is not a valid member of %s' % (
```

Comment:
We could return the AttributeError, my initial tinkering suggested it wasn't wildly helpful and that a better error messages was. Without this wrapping, you get something like:

```
>>> import importlib
>>> module = importlib.import_module('hashlib')
>>> getattr(hashlib, 'foo')
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
AttributeError: 'module' object has no attribute 'foo'
>>> 
```

"module" is totally not helpful here. With my changes, you get something like:

```
ValueError: 'foo' is not a valid member of hashlib'
```

I thought about wrapping the ImportError and then decided not to because the ImportError itself is more informative than what I'd probably do if wrapping it.